tensorflow/python/kernel_tests/draw_bounding_box_op_test.py

```diff
@@ -85,9 +90,12 @@ def _testDrawBoundingBoxColorCycling(self, img):
       image = ops.convert_to_tensor(image)
       image = image_ops_impl.convert_image_dtype(image, dtypes.float32)
       image = array_ops.expand_dims(image, 0)
-      image = image_ops.draw_bounding_boxes(image, bboxes)
+      if colors is None:
```

Comment:
This shouldn't be needed. We should be testing the `draw_bounding_boxes` function exposed in the API instead of testing the generated wrappers. So, can't this be:

```python
image = image_ops.draw_bounding_boxes(image, bboxes, colors=colors_arg)
```
?

Where `colors_arg` points to the `colors` value provided to `_testBoundingBoxColorCycling` (i.e., before it was set to `default_color_table` to generate `test_drawn_image`?

Reply:
Updated.